src/Microsoft.PowerShell.SHiPS/Resources/Resource.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These are auto-generated code. Can you please revert it?

Reply:
didn't get you, is it about reverting the whole change in Resource.resx file

Reply:
Yes please revert all changes made in this file .resx  file.